extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -1762,7 +1762,7 @@ describes.realWin(
       });
     });
 
-    describe('parentPostMessage in inabox case', () => {
+    describe('parentPostMessage', () => {
```

Comment:
Can we please add one more test to cover the case that `parentPostMessage` is supported in FIE. Thanks

Reply:
done.